package.json

```diff
@@ -69,23 +66,22 @@
     "oas-validator": "^5.0.0",
     "openapi-backend": "^5.9.1",
     "redis": "^4.1.0",
-    "reflect-metadata": "^0.1.13",
-    "ts-node": "^10.8.0",
-    "tsconfig-paths": "^4.0.0",
```

Comment:
Don't we need ts path support?

Reply:
I don't see any reason why it's needed

Reply:
Ok, that's surprising. Without having that in dependencies, we can still import? Anyway if you are sure, lets resolve this comment.